specification/network/resource-manager/Microsoft.Network/stable/2020-08-01/networkManagerSecurityConfiguration.json

```diff
@@ -0,0 +1,1349 @@
+          "items": {
+            "type": "string",
+            "description": "The destination port."
+          },
+          "description": "The destination port ranges."
+        },
+        "access": {
+          "$ref": "#/definitions/SecurityConfigurationRuleAccess",
+          "description": "Indicates the access allowed for this particular rule"
+        },
+        "priority": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The priority of the rule. The value can be between 1 and 4096. The priority number must be unique for each rule in the collection. The lower the priority number, the higher the priority of the rule."
+        },
+        "direction": {
+          "$ref": "#/definitions/SecurityConfigurationRuleDirection",
+          "description": "Indicates if the traffic matched against the rule in inbound or outbound."
+        },
+        "provisioningState": {
+          "readOnly": true,
+          "$ref": "./network.json#/definitions/ProvisioningState",
+          "description": "The provisioning state of the security Configuration resource."
+        }
+      },
+      "required": [
+        "protocol",
+        "direction"
+      ],
+      "description": "Security rule resource."
+    },
+    "SecurityConfigurationRuleAccess": {
+      "type": "string",
+      "description": "Whether network traffic is allowed or denied.",
+      "enum": [
+        "Allow",
+        "Deny",
+        "AlwaysAllow"
+      ],
+      "x-ms-enum": {
+        "name": "SecurityConfigurationRuleAccess",
+        "modelAsString": true
+      }
+    },
+    "SecurityConfigurationRuleDirection": {
+      "type": "string",
+      "description": "The direction of the rule. The direction specifies if rule will be evaluated on incoming or outgoing traffic.",
+      "enum": [
+        "Inbound",
+        "Outbound"
+      ],
+      "x-ms-enum": {
+        "name": "SecurityConfigurationRuleDirection",
+        "modelAsString": true
+      }
+    },
+    "NetworkManagerSecurityGroupItem": {
+      "type": "object",
+      "properties": {
+        "networkGroupId": {
+          "type": "string",
+          "description": "Network manager group Id."
+        }
+      },
+      "description": "Network Manager Security Group Item."
+    },
+    "AddressPrefixItem": {
+      "type": "object",
+      "properties": {
+        "addressPrefix": {
+          "type": "string",
+          "description": "Address Prefix."
+        },
+        "addressPrefixType": {
+          "type": "string",
+          "description": "Address Prefix Type.",
+          "enum": [
+            "IPPrefix",
+            "ServiceTag"
+          ],
+          "x-ms-enum": {
+            "name": "AddressPrefixType",
+            "modelAsString": true
+          }
+        }
+      },
+      "description": "Address Prefix Item."
+    },
+    "NetworkManagerSecurityConfigurationImport": {
+      "properties": {
+        "networkSecurityGroupURI": {
```

Comment:
use name networkSecurityGroupUris as it's an array. Also usage of name URI here and in adminSecurityConfigurationUri should be in same casing for consistency.